types/node/test.d.ts

```diff
@@ -1007,12 +1007,20 @@ declare module "node:test" {
          */
         restore(): void;
     }
-    type Timer = "setInterval" | "clearInterval" | "setTimeout" | "clearTimeout";
+    type Timer = "setInterval" | "clearInterval" | "setTimeout" | "clearTimeout" | "setImmediate" | "Date";
```

Comment:
https://github.com/nodejs/node/blob/c89bae161b0e36c83ab07f09ed18b5da97285fec/lib/internal/test_runner/mock/mock_timers.js#L64

I don't see `'clearInterval'` and `'clearTimeout` in list of supported timers.

In docs:
> The currently supported timer values are `'setInterval'`, `'setTimeout'`, `'setImmediate'`, and `'Date'`.

Reply:
You are absolutely correct, I didn't change the previous ones as I assumed they were correct, just added new ones, will remove it